groups/sig-scalability/groups.yaml

```diff
@@ -86,6 +86,28 @@ groups:
       - wojtekt@google.com
       - zawodny@google.com
 
+  # This group represents privileged access to a staging project,
+  # allowing the members to directly write to GCS and GCR within the
+  # project, as well as trigger Cloud Build within the project. Ideally
+  # this level access is used solely for troubleshooting purposes.
+
+  - email-id: k8s-infra-staging-perf-tests@kubernetes.io
+    name: k8s-infra-staging-perf-tests
+    description: |-
+      ACL for staging perf-tests artifacts
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - azylinski@google.com
+      - jkaniuk@google.com
+      - jprzychodzen@google.com
+      - jupblb@google.com
+      - maciejborsz@google.com
+      - marcelzieba@google.com
+      - shyam123.jvs95@gmail.com
+      - wojtekt@google.com
+      - zawodny@google.com
```

Comment:
@marseel non-blocking comment: you can also use the scalability on-call ML : https://github.com/kubernetes/k8s.io/blob/main/groups/sig-scalability/groups.yaml#L45-L63. main benefit is you just maintain one list of members.